### PR TITLE
Add Cli command to create custom cli commands

### PR DESCRIPTION
**Pull request by Lukicdev**
# Description

Add WP CLI command to generate custom CLI command in project 

# Screenshots / Videos

https://user-images.githubusercontent.com/47175420/147069665-60bc12e9-faf9-4835-9cfc-033472464699.mp4

### CODE REVIEW

**Comment by dingo-d** [comment]
lol

I just couldn't pass the opportunity 😂 

Can you just check why the phpstan is failing and why the codecov marked this as a failure?

File: src/CliCommands/AbstractCustomCommand.php [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+namespace EightshiftLibs\CliCommands;
+
+use EightshiftLibs\Cli\CliHelpers;
+use EightshiftLibs\Services\ServiceInterface;
+
+/**
+ * Abstract base custom command class
+ */
+abstract class AbstractCustomCommand implements ServiceInterface
```
Comment (dingo-d) [inline src/CliCommands/AbstractCustomCommand.php:19]: Why is this class needed? There already is an `AbstractCli` class that handles command registration? Is there anything different that this abstract class does from AbstracCli one? 

If yes, can't this different part be included in an AbstractCli class?

**Review by dingo-d — CHANGES_REQUESTED** [review]
Left one question, good job otherwise 👍🏼

**Review by dingo-d — APPROVED** [review]
Great job!

**Review by goranalkovic-infinum — APPROVED** [review]

**Pull request merged**